whois-nrtm4/src/main/java/net/ripe/db/nrtm4/dao/NrtmVersionInfoRepository.java

```diff
@@ -113,12 +129,12 @@ INSERT INTO version_info (source_id, version, session_id, type, last_serial_id,
                 return pst;
             }, keyHolder
         );
-        return new NrtmVersionInfo(keyHolder.getKeyAs(Long.class), source, version, sessionID, type, lastSerialId, now);
+        return new NrtmVersionInfo(keyHolder.getKeyAs(Long.class), sourceID, source, version, sessionID, type, lastSerialId, now);
     }
 
     public NrtmVersionInfo findLastSnapshotVersion(final NrtmSource source) {
         final String sql = """
```

Comment:
If we are creating snapshot file for all sources at once, do we really need this method?